pkg/cosign/verify.go

```diff
@@ -275,6 +301,25 @@ func (sp *SignedPayload) VerifyClaims(d *v1.Descriptor, ss *payload.SimpleContai
 	return nil
 }
 
+func (sp *SignedPayload) VerifyBundle() (bool, error) {
```

Comment:
should probably just return `error`, unless `error == nil` would ever not imply a successful validation

Reply:
if Bundle isn't available for some reason, having both tells us that it was unverified but also there was no actual error to return. The bool is used later to decide if we still need to hit the tlog.

Reply:
Ah, I just noticed the very next line. Carry on :p